www/grpc/transaction.go

```diff
@@ -191,6 +229,7 @@ func (s *transactionServer) GetRawUnbondTransaction(_ context.Context,
 	}, nil
 }
 
+// Deprecated: GetRawWithdrawTransaction.
```

Comment:
What does this comment want to say?

Reply:
This comment says this message, method or function is deprecated.